src/html/partials/behind-the-scenes.html

```diff
@@ -1,4 +1,57 @@
 <section class="behind-scenes-section">
+
   <h2 class="sm-title">Behind The Scenes</h2>
-  <p class="bg-intro">While Florence is only a concept, here are some things that will be helpful in making her come alive.</p>
-</section>
+  <p class="lg-text">While Florence is only a concept, here are some things that will be helpful in making her come alive.</p>
+
+  <!-- <div class="behind-scenes-section__florence">
+    <div class="behind-scenes-section__florence__book"></div>
+    <div class="behind-scenes-section__florence__gear"></div>
+    <div class="behind-scenes-section__florence__comment"></div>
+    <div class="behind-scenes-section__florence__heart"></div>
+    <div class="behind-scenes-section__florence__link"></div>
+  </div> -->
+
+  <ul class="behind-scenes-section__list">
+    <li class="behind-scenes-section__list__item">
+      <button class="behind-scenes-section__list__item__btn--book"></button>
+      <div class="behind-scenes-section__list__item__wrapper">
+        <h3 class="sm-list-title -orange">Medical Knowledge Bases</h3>
+        <p class="sm-sans-serif-text">Side effects and drug interactions are currently available in services like <a href="https://www.epocrates.com/" target="_blank">Epocrates</a> and <a href="https://www.medscape.com/" target="_blank">Medscape</a>. Though designed for long-form reading, it could be pruned and adapted to fit messaging constraints.</p>
+      </div>
+    </li>
+
+    <li class="behind-scenes-section__list__item">
+      <button class="behind-scenes-section__list__item__btn--gear"></button>
+      <div class="behind-scenes-section__list__item__wrapper">
+        <h3 class="sm-list-title -orange">Algorithmic Content</h3>
+        <p class="sm-sans-serif-text">Generating believable messages that don’t feel artificial or templated is key to maintaining the illusion. Tools like <a href="https://automatedinsights.com/wordsmith/" target="_blank">Automated Insights Wordsmith</a> hint to possibilities where content can be produced on the fly and personalized to each user.</p>
+      </div>
+    </li>
+
+    <li class="behind-scenes-section__list__item">
+      <button class="behind-scenes-section__list__item__btn--comment"></button>
+      <div class="behind-scenes-section__list__item__wrapper">
+        <h3 class="sm-list-title -orange">Natural Language Processing</h3>
+        <p class="sm-sans-serif-text">Florence’s conversational style will encourage users to speak to her like a friend. Natural language processing, like those offered by <a href="https://www.youtube.com/watch?v=r7E1TJ1HtM0" target="_blank">IBM’s Watson</a>, will be needed to extract meaning and respond intelligently.</p>
```

Comment:
💯 for curly quotes! (Florence’s)

Reply:
Where am I missing this?

Reply:
You're not missing anything. I was saying you get imaginary points for using the right kind of quotes! 👍